src/test/groovy/com/bmuschko/gradle/docker/internal/RegistryAuthLocatorTest.groovy

```diff
@@ -214,7 +219,15 @@ class RegistryAuthLocatorTest extends Specification {
         File configFile = new File(getClass().getResource(CONFIG_LOCATION + configName).toURI())
         RegistryAuthLocator locator
         if (mockHelper) {
-            String command = configFile.getParentFile().getAbsolutePath() + '/docker-credential-'
+            String command = null
+            if (isWindows()) {
+                def matcher = configFile.getParentFile().getAbsolutePath() =~ /(?<drive>[A-Z]):.*/
+                matcher.matches()
+                def driveLetter = matcher.group('drive')
+                command = 'C:\\msys64\\usr\\bin\\bash /' + (configFile.getParentFile().getAbsolutePath().toString().replace(driveLetter + ':', driveLetter.toLowerCase()).replace('\\', '/')) + '/docker-credential-'
+            } else {
+                command = configFile.getParentFile().getAbsolutePath() + '/docker-credential-'
```

Comment:
I think this would be expressed better as `new File(configFile.getParentFile(), 'docker-credential-').getAbsolutePath()`.

Reply:
done